cfme/tests/test_rest.py

```diff
@@ -239,6 +252,59 @@ def automation_requests_data():
     } for index in range(1, 5)]
 
 
+@pytest.fixture(scope='function')
+def roles(request, rest_api):
+    if "create" not in rest_api.collections.roles.action.all:
+        pytest.skip("Create roles action is not implemented in this version")
```

Comment:
If this "version" is tied to an appliance version, for example 5.4, we should also wrap the test with an uncollectif to drop the test from being run if it's on 5.4